Modified the TRB init method to create receivers directly

bieryAtFnal opened the pull request:
…for the `data_fragment_all` and `mon_connection` local names without looping over all conn_refs

bieryAtFnal committed:
Modified the TRB init method to create receivers for data_fragment_all and mon_connection local names without looping over all conn_refs

eflumerf commented on plugins/TriggerRecordBuilder.cpp:
```diff
@@ -69,31 +69,16 @@ TriggerRecordBuilder::init(const data_t& init_data)
-    } else if (ref.name == "mon_connection") {
-      m_mon_receiver = iom->get_receiver<dfmessages::TRMonRequest>(ref.uid);
-    }
+  m_fragment_inputs.push_back(iom->get_receiver<std::unique_ptr<daqdataformats::Fragment>>(ci["data_fragment_all"]));
```
Can we change this member to `m_fragment_input`, with appropriate vector-to-member type change?

eflumerf committed:
Since we have a single Fragment input connection (and have had just one
for quite a while now), remove potentially-confusing multi-source
member/logic.

bieryAtFnal committed:
Merge remote-tracking branch 'origin/develop' into kbiery/trb_data_fragment_all_change

bieryAtFnal committed:
Merge branch 'kbiery/trb_data_fragment_all_change' into eflumerf/SingleFragmentInput

bieryAtFnal committed:
Merge pull request #270 from DUNE-DAQ/eflumerf/SingleFragmentInput

Since we have a single Fragment input connection (and have had just one

roland-sipos reviewed (approved):
LGTM!
